core/src/templates/components/link/link.twig

```diff
@@ -19,4 +19,4 @@
  *
  */
 #}
-<a{{ attributes }} href="{{ link_href }}" class="su-link {{ modifier_class }}">{{ link_content }}</a>
+<a {{ attributes }} href="{{ link_href }}" class="su-link {{ modifier_class }}">{{ link_content }}</a>
```

Comment:
In >90% of our templates, we put a space in the `attributes`' value, and put no space between the tag and `{{ attributes }}` in the `.twig` file. I think we should stay consistent.

Reply:
The Drupal convention was to have no space but it is causing problems to do it that way in other systems whereas having the space causes no extra problems in Drupal. I say we add the space.